app/assets/javascripts/langscope.js

```diff
@@ -0,0 +1,57 @@
+var map;
+
+$(document).ready(function() {
+
+    var countryName = document.getElementById('country-name');
+    var speakers = document.getElementById('speakers');
+
+    $.get("/countries.json", function(data) {
+
+    var countryCollection = {};
+
+    for (var i = 0; i < data.length; i++) {
+        countryCollection[data[i].code] = {
```

Comment:
I think I mentioned it before, we prefer 2 space indentation for all of our apps. I would strongly suggest to start using that instead. You can convert your tab to be two spaces.